src/commands/common.ts

```diff
@@ -159,3 +159,22 @@ export const UrlFolder: Type<string, URL> = {
     return url;
   },
 };
+
+/**
+ * Remove a trailing 'm' from a input value and validate the input is a number.
+ *
+ * @param str input value
+ * @returns value without trailing 'm' (if it exists)
+ * @throws if input is not a valid number
+ */
+export const MeterAsString: Type<string, string> = {
+  from(str) {
+    const gsd = str.endsWith('m') ? str.slice(0, -1) : str;
+
+    if (isNaN(Number(gsd))) {
+      throw new Error(`Invalid GSD value: ${gsd}. GSD must be a number.`);
```

Comment:
```suggestion
      throw new Error(`Invalid meters value: ${meters}. Must be a number.`);
```

Reply:
I believe the tests look for this exact string :sob:

Reply:
Oh! That was just to align with the function becoming more "generic" that only `gsd`